client/knowledge_base_search/src/containers/QuestionBarContainer/QuestionBarContainer.jsx

```diff
@@ -40,6 +40,10 @@ class QuestionBarContainer extends Component {
       offset,
     } = this.props;
 
+    function isCurrentQuestion(question) {
```

Comment:
i would move this up alongside the `handleX` functions like:
```
isCurrentQuestion(question) {
  return question === this.props.currentQuery;
}
```

then call it with
`this.isCurrentQuestion(query.question)`